compose-default.yml

```diff
@@ -62,8 +62,8 @@ services:
       # recommended, as it creates a dependency on the host environment.  
       # consider creating a more formal layout instead.
       #
-      - ${PATH_APP}/web:/app/web 
-      - ${PATH_APP}/vendor:/app/vendor
+      - ${PATH_SOURCE}/web:/app/web 
+      - ${PATH_SOURCE}/vendor:/app/vendor
```

Comment:
The docker composer now looks for source in PATH_SOURCE, which is the variable that has been set to point to where source code can be found,  PATH_APP is used as the root path of the project.